lib/web_catalog/post_processors.es6.js

```diff
@@ -163,11 +163,48 @@ foam.CLASS({
 });
 
 foam.CLASS({
+  name: 'CopyToInterfaceProcessor',
+  package: 'org.chromium.apis.web',
+  implements: ['org.chromium.apis.web.ApiCatalogPostProcessor'],
+
+  documentation: 'Copy APIs from one interface to another.',
```

Comment:
Document that things in fromInterfaceName will only be copied if they aren't already in toInterfaceName